### PR TITLE
West Midlands Class 3 _ Pezhman_Azizi_Git&GitHub_Week1

### PR DESCRIPTION
**Pull request by Pezhman-Azizi**
I corrected the wrong answers as best of my understanding 

# Your Details

- Your Name: Pezhman Azizi
- Your City: Birmingham
- Your Slack Name: Pezhman Azizi

# Homework Details

- Module: Git & GitHub
- Week: 2

-----
[View rendered homework.md](https://github.com/Pezhman-Azizi/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -4,77 +4,82 @@
...
 
-You delete it
+You will create a version on your local computer without any changes on the actual project on Github. Forking gives a chance to manipulate the codes before sending in on the web to every one see it. 
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+It will create a folder in youe local computer.
```
[the remark this answers is not in the document]
Comment (Pezhman-Azizi) [reply homework.md:70]: Thanks @smkamaldar for your comment

File: homework.md [hunk 2]
```diff
@@ -4,77 +4,82 @@
...
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1. When people want to share their code to each other they can put it on Github.
+
+2. Benifited with "commit" option you are able to see when and what change has been applied to the project on your team.
+
+3. Using Github you do not need to be at the same place working on your project at the same time as it an Online app. You can have worldwide team mates :D .
```
[the remark this answers is not in the document]
Comment (Pezhman-Azizi) [reply homework.md:57]: Thanks that is very kind of you :)

File: homework.md [hunk 3]
```diff
@@ -92,22 +97,22 @@ A place for us to store and share our code.
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+It kind of saves the changes on you local version of the code.
```
Comment (GemPearl) [inline homework.md:100]: Thats correct. It worth including that it saves at a given time. Thus it is a snapshot.

File: homework.md [hunk 4]
```diff
@@ -92,22 +97,22 @@ A place for us to store and share our code.
...
 
-Microsoft Word
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To save what changes you made to the code
```
Comment (GemPearl) [inline homework.md:113]: Do you mean  "To say"?

**Comment by Haseeb3927** [comment]
Great work Pezhman 👍  keep it up

**Pull request closed without merging**